spec/featureSpec.js

```diff
@@ -0,0 +1,108 @@
+// implement some sort of user interface with jquery - needs:
+  // index.html, interface.html, index.css
+// final frame
+// all strikes - 300points
+
+describe ('bowling', function() {
+
+  var game;
+
+  beforeEach(function () {
+    game = new Game();
+  });
+
+  describe ('for a gutter game', function() {
+    it('returns 0', function() {
+      for (let i = 0; i < 20; i++) {
+        game.roll(0);
+      }
+      expect(game.gameover()).toEqual(0);
+    });
+  });
+
+  describe ('one frame, no spares/strikes', function() {
+    it('returns total of frame', function() {
+      game.roll(3);
+      game.roll(4);
+      expect(game.gameover()).toEqual(7);
+    });
+  });
+
+  describe('multiple frames, no spares/strikes', function() {
+    it('resets frame after every 2 rolls', function() {
+      game.roll(3);
+      game.roll(4);
+      expect(game.currentFrame.pins).toEqual([]);
```

Comment:
This assertion is testing the state of the program rather than the behaviour for the end user: https://github.com/makersacademy/skills-workshops/blob/master/practicals/testing/behaviour_not_state.md

Reply:
Thanks for the link to the practical, this is something I've not really focused on yet so I will make it my focus for upcoming projects.